build/build.proj

```diff
@@ -108,7 +108,7 @@
 
   </Target>
 
-  <Target Name="Build" DependsOnTargets="RestorePackages;BuildSolution;BuildNuGetPackages;Test" />
-  <Target Name="Rebuild" DependsOnTargets="RestorePackages;RebuildSolution;RebuildNuGetPackages;Test" />
+  <Target Name="Build" DependsOnTargets="RestorePackages;BuildSolution;BuildNuGetPackages;BuildModernVsixPackages;Test" />
```

Comment:
Additionally, this makes the build.cmd switch (modernvsixonly)[https://github.com/dotnet/roslyn-project-system/blob/master/build.cmd#L69] redundant - if Microbuild has fixed their bug and you able to merge this PR, then you should remove that switch.